great_expectations/profile/basic_dataset_profiler.py

```diff
@@ -73,30 +73,22 @@ def _get_column_cardinality(cls, df, column):
 
         return cardinality
 
-    @classmethod
-    def _get_value_set(cls, df, column):
-        partition_object = {
-            "values": ["GE_DUMMY_VAL"],
-            "weights": [1.0]
-        }
-
-        df.expect_column_kl_divergence_to_be_less_than(column, partition_object=partition_object,
-                                                   threshold=0, result_format='COMPLETE')
-
     @classmethod
     def _profile(cls, dataset):
 
 
         df = dataset
 
+        df.expect_table_row_count_to_be_between(min_value=0, max_value=None)
+        df.expect_table_columns_to_match_ordered_list(None)
+
         for column in df.get_table_columns():
-            df.expect_column_to_exist(column)
+            # df.expect_column_to_exist(column)
 
             type_ = cls._get_column_type(df, column)
             cardinality= cls._get_column_cardinality(df, column)
-            df.expect_column_values_to_not_be_null(column, mostly=0)
-            df.expect_column_values_to_be_in_set(
-                column, [], result_format="SUMMARY")
+            df.expect_column_values_to_not_be_null(column, mostly=0.5) # The renderer will show a warning for columns that do not meet this expectation
+            df.expect_column_values_to_be_in_set(column, [], result_format="SUMMARY")
```

Comment:
Comment here to flag that this is the expectation where we expect it to fail?

Reply:
Yes, we need to set discard_failed_expectations to make sure that, even it fails, the expectation is preserved.